### ISSUE TITLE
Favicon does not use kit `paths`

### ISSUE THREAD

**Issue by DblK** [sveltejs/kit#3231]
### Describe the bug

When setting the following configuration for svelete:

```
...
kit: {
  ...
  paths: {
    base: '/admin'
  },
}
```

In the `app.html` there is the following line `<link rel="icon" href="/favicon.png" />`.
The `%svelte.head%` add the path while generating files but favicon is left untouched.

### Reproduction

Simply create a new app and change the kit config as previously stated.

### System Info

```shell
System:
    OS: macOS 10.15.7
    CPU: (8) x64 Intel(R) Core(TM) i7-4870HQ CPU @ 2.50GHz
    Memory: 18.91 MB / 16.00 GB
    Shell: 5.7.1 - /bin/zsh
  Binaries:
    Node: 14.17.0 - ~/.nvm/versions/node/v14.17.0/bin/node
    npm: 7.24.0 - ~/.nvm/versions/node/v14.17.0/bin/npm
  Browsers:
    Chrome: 96.0.4664.110
    Firefox: 89.0.2
    Safari: 14.0.3
  npmPackages:
    @sveltejs/adapter-auto: next => 1.0.0-next.9 
    @sveltejs/adapter-static: ^1.0.0-next.24 => 1.0.0-next.24 
    @sveltejs/kit: next => 1.0.0-next.216 
    svelte: ^3.44.0 => 3.44.3
```

### Severity

annoyance

### Additional Information

To solve the issue I have to manually change the path for the favicon.
Since I do not know yet where I will serve my app, I will need to change the svelte config BUT also the `app.html` to avoid 404 error for favicon.

**Comment by ghostdevv** [comment]
Very interesting to see where this issue will go, as app.html isn't processed and talks of potentially doing that occurred in #3127

I wonder what happens when you import a asset in one of your routes such as `<img src="/favicon.png" />` if that image is also broken or if that url is processed

**Comment by Conduitry** [comment]
I would suggest just putting the `<link rel=icon>` somewhere else where you do have access to the `base` information - like `__layout.svelte`, and then using https://kit.svelte.dev/docs#modules-$app-paths to construct the appropriate URL. I don't think it makes sense to make it any more complicated than that or to process `app.html` in some way.

I could go either way on whether the `create-svelte` template apps should be updated like this.

**Comment by Rich-Harris** [comment]
This will be fixed by #3234 — just add `%svelte.assets%` to the template (i.e. `%svelte.assets%/favicon.png`)
